.github/workflows/continuous-integration-workflow.yml

```diff
@@ -61,6 +61,17 @@ jobs:
       - name: "extract ct-ng"
         run: |
           tar -xf ct-ng.tar
+      - name: download tarballs
+        uses: actions/download-artifact@v3
+        continue-on-error: true
+        with:
+          name: src.tar
```

Comment:
Will this work? The thing that creates src.tar hasn't run yet so I don't think it's smart enough to use the one from the last build.

Reply:
You seem to be right. I wanted to test this because artifact last for 90 days und caches only for 7.